### PR TITLE
Add information (new icons) about hosts update

### PR DESCRIPTION
**Pull request by smau-centreon**
## Description

Add information (new icons) about hosts update - 22.10

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [x] 22.10.x (next)

### CODE REVIEW

File: versioned_docs/version-22.10/monitoring/discovery/hosts-discovery.md [hunk 1]
```diff
@@ -153,27 +153,34 @@ next to the status of a **Finished** job to visualize the results.
...
+  The results show the discovered hosts. Icons allow you to identify the action that will be performed on the host:
+   
+   - The **plus** icon means the host does not exist yet and will be created.
+   - The **arrow** icon means the host already exists and its configuration will be updated.
 
-    The results will be updated automatically in the preview once you click on the **Save** icon at the top right of the panel.
+2. Select the hosts you want to add to the list of monitored hosts and click the **Save** button. Then click the **Back** button to return to the **Discovery** page.
 
-    - If the job was set to **Manual analysis** at step 5 of the wizard, bear in mind that editing mappers after a discovery job has been executed only makes sense if the hosts have not yet been added to your monitoring configuration.
+3. If you want, you can edit the mappers linked to this job. Click on the job to display its properties.  In the **Mappers** section, edit the mapper with its **Edit** icon: ![image](../../assets/monitoring/discovery/host-discovery-edit.png#thumbnail1)
```
Comment (omercier) [inline versioned_docs/version-22.10/monitoring/discovery/hosts-discovery.md:163]:
```suggestion
3. If you want, you can edit the mappers linked to this job. Click on the job to display its settings.  In the **Mappers** section, edit the mapper with its **Edit** icon: ![image](../../assets/monitoring/discovery/host-discovery-edit.png#thumbnail1)
```

File: versioned_docs/version-22.10/monitoring/discovery/hosts-discovery.md [hunk 2]
```diff
@@ -153,27 +153,34 @@ next to the status of a **Finished** job to visualize the results.
...
 
-    - If the job was set to **Automatic analysis** at step 5 of the wizard and **Update existing hosts** was selected, the hosts will be updated when you re-run the job (see [Edit a discovery job](#edit-a-discovery-job)).
+  The results will be updated automatically in the preview once you click on the **Save** icon at the top right of the panel.
+  
+   - If the job was set to **Manual analysis** at step 5 of the wizard, bear in mind that editing mappers after a discovery job has been executed only makes sense if the hosts have not yet been added to your monitoring configuration.
```
Comment (omercier) [inline versioned_docs/version-22.10/monitoring/discovery/hosts-discovery.md:167]: This warning makes no sense now that the hosts can be updated in manual mode ;-)
I suggest removing it

Comment (omercier) [inline versioned_docs/version-22.10/monitoring/discovery/hosts-discovery.md:167]:
```suggestion
```

File: versioned_docs/version-22.10/monitoring/discovery/hosts-discovery.md [hunk 3]
```diff
@@ -153,27 +153,34 @@ next to the status of a **Finished** job to visualize the results.
...
+   
+   - If the job was set to **Automatic analysis** at step 5 of the wizard and **Update existing hosts** was selected, the hosts will be updated when you re-run the job (see [Edit a discovery job](#edit-a-discovery-job)).
 
-3. If your job was set to **Manual analysis** at step 5 of the wizard, select the hosts you want to add to the configuration and click on the **Save**
+4. If your job was set to **Manual analysis** at step 5 of the wizard, select the hosts you want to add to the configuration and click on the **Save**
```
Comment (omercier) [inline versioned_docs/version-22.10/monitoring/discovery/hosts-discovery.md:171]:
```suggestion
4. If your job was set to **Manual analysis** at step 5 of the wizard, select the hosts you want to add or update in the configuration and click on the **Save**
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/monitoring/discovery/hosts-discovery.md [hunk 4]
```diff
@@ -161,30 +161,37 @@ Plusieurs actions peuvent être réalisées sur les tâches :
...
+2. Sélectionnez les hôtes que vous souhaitez ajouter à la liste des hôtes supervisés et cliquez sur le bouton **Sauvegarder**. Cliquez ensuite sur le bouton **Retour** pour revenir à la page **Découverte**.
+
+3. Si vous le souhaitez, vous pouvez éditer les [modificateurs](#comment-utiliser-les-modificateurs) liés à la tâche. Cliquez sur la tâche pour afficher ses propriétés. Dans la section **Modificateurs**, modifiez le modificateur en cliquant sur le bouton d'édition : ![image](../../assets/monitoring/discovery/host-discovery-edit.png#thumbnail1)
 
     Les changements seront appliqués directement à la liste des résultats lorsque vous cliquerez sur l'icône **Sauvegarder** en haut à droite du panneau.
 
     - Si vous aviez sélectionné **Analyse manuelle** à l'étape 5 de l'assistant, attention : éditer les modificateurs après l'exécution de la tâche de découverte n'a de sens que si les hôtes n'ont pas encore été ajoutés à la configuration. Les changements de modificateurs au niveau de la tâche de découverte ne sont pas pris en compte pour les hôtes déjà créés.
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/monitoring/discovery/hosts-discovery.md:177]: This warning makes no sense now that the hosts can be updated in manual mode ;-)
I suggest removing it

Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/monitoring/discovery/hosts-discovery.md:177]:
```suggestion
```

File: versioned_docs/version-22.10/getting-started/autodisco-aws.md [hunk 5]
```diff
@@ -75,6 +75,11 @@ You must have:
 
     ![image](../assets/getting-started/aws-results.png)
 
+    Icons allow you to identify the action that will be performed on the host:
+   
+   - The **plus** icon means the host does not exist yet and will be created.
+   - The **arrow** icon means the host already exists and its configuration will be updated.
```
[the remark this answers is not in the document]
Comment (smau-centreon) [reply versioned_docs/version-22.10/getting-started/autodisco-aws.md:81]: If we add this note, this forces us to specify all types of mappers not taken into account. Can we provide this list?

[the remark this answers is not in the document]
Comment (omercier) [reply versioned_docs/version-22.10/getting-started/autodisco-aws.md:81]: Actually only Property mappers are ignored.

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/monitoring/discovery/hosts-discovery.md [hunk 6]
```diff
@@ -161,30 +161,36 @@ Plusieurs actions peuvent être réalisées sur les tâches :
...
 1. À la page **Configuration > Hôtes > Découverte**, cliquez sur la flèche ![image](../../assets/monitoring/discovery/host-discovery-goto-results.png) à côté du statut d'une tâche terminée pour en visualiser le résultat.
 
-    ![image](../../assets/monitoring/discovery/host-discovery-hosts-listing.png)
+   ![image](../../assets/monitoring/discovery/host-discovery-hosts-listing.png)
+   
+   La liste de résultats montre les hôtes découverts. Des icônes vous permettent d'identifier l'action qui sera effectuée sur l'hôte :
+   
+   - L'icône **plus** signifie que cet hôte n’existe pas encore et sera créé.
+   - L'icône **flèche** signifie que cet hôte existe déjà et sa configuration sera mise à jour.
+   > Notez que tous les modificateurs de type **Propriété** ne sont pas pris en compte lors du processus de mise à jour.
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/monitoring/discovery/hosts-discovery.md:170]:
```suggestion
   > Notez que les modificateurs de type **Propriété** ne sont pas pris en compte lors du processus de mise à jour.
```

Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs/version-22.10/monitoring/discovery/hosts-discovery.md:170]: "tous les ... ne sont pas" => peut laisser penser que certains le sont alors qu'aucun ne l'est

File: versioned_docs/version-22.10/monitoring/discovery/hosts-discovery.md [hunk 7]
```diff
@@ -153,27 +153,33 @@ next to the status of a **Finished** job to visualize the results.
 
   ![image](../../assets/monitoring/discovery/host-discovery-hosts-listing.png)
 
-2. If you want, you can edit the mappers linked to this job by clicking the **Edit** icon: ![image](../../assets/monitoring/discovery/host-discovery-edit.png#thumbnail1)
+  The results show the discovered hosts. Icons allow you to identify the action that will be performed on the host:
+   
+   - The **plus** icon means the host does not exist yet and will be created.
+   - The **arrow** icon means the host already exists and its configuration will be updated.
+   > Note that all the **Property** mappers are not taken into account during the update process.
```
Comment (omercier) [inline versioned_docs/version-22.10/monitoring/discovery/hosts-discovery.md:160]:
```suggestion
   > Note that the **Property** mappers are not taken into account during the update process.
```

Comment (omercier) [reply versioned_docs/version-22.10/monitoring/discovery/hosts-discovery.md:160]: Same as in French. If you want to emphasis, then let's say "none of the"?

**Review by omercier — APPROVED** [review]

**Review by sadikraniya — APPROVED** [review]

**Pull request merged**